### ISSUE TITLE
Default is ignored on lists

### ISSUE THREAD

**Issue by martenjacobs** [CITGuru/PyInquirer#17]
It seems that the `default`-key is ignored on list-type questions

Consider the following code:
```python
from PyInquirer import prompt, print_json

answer = prompt({
        'type': 'list',
        'name': "test-question",
        'message': "Select item",
        'default': 2,
        'choices': [
            {   "name"  : "Item 1",
                "value" : "item1"   },
            {   "name"  : "Item 2",
                "value" : "item2"   },
            {   "name"  : "Item 3",
                "value" : "item3"   },
            {   "name"  : "Item 4",
                "value" : "item4"   },
        ]
    })

print_json(answer)
```

### Expected result:
`Item 3` is initially selected in the prompt

### Actual result:
`Item 1` is initially selected in the prompt

### Environment:
OS: macOS 10.14 Mojave
```bash 
$ python3 --version
Python 3.6.1
```
```bash 
$ python3 -m pip show PyInquirer
Name: PyInquirer
Version: 1.0.2
Summary: A Python module for collection of common interactive command line user interfaces, based on Inquirer.js
Home-page: https://github.com/CITGuru/PyInquirer/
Author: Oyetoke Toby
Author-email: oyetoketoby80@gmail.com
License: MIT
Location: /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
Requires: prompt-toolkit, Pygments, regex
Required-by:
```

**Comment by CITGuru** [comment]
I'll take a closer look at this.

**Comment by CITGuru** [comment]
Yes its actually ignored cos, you have to choose one from the list.

**Comment by martenjacobs** [comment]
From documentation it appears you should be able to define the initially selected list item. Either the documentation should be clarified or this feature should be implemented.

**Comment by ajdavis** [comment]
I believe this issue should be reopened, it's still present in version 1.0.3 on PyPI and on master.

**Comment by ewels** [comment]
Looks like this has now been fixed in #68 - we just need a release (#90)
